src/Stack/PackageDump.hs

```diff
@@ -64,7 +64,8 @@ ghcPkgCmdArgs pkgexe@(GhcPkgExe pkgPath) cmd mpkgDbs sink = do
     case reverse mpkgDbs of
         (pkgDb:_) -> createDatabase pkgexe pkgDb -- TODO maybe use some retry logic instead?
         _ -> return ()
-    sinkProcessStdout (toFilePath pkgPath) args sink'
+    (_,res) <- sinkProcessStderrStdout (toFilePath pkgPath) args CL.sinkNull sink'
+    pure res
```

Comment:
My own version of this was:
~~~
snd <$> sinkProcessStderrStdout (toFilePath pkgPath) args CL.sinkNull sink'
~~~
It has the advantage (or disadvantage, depending on your perspective) of being a single line.

Reply:
I have no strong opinions either wise. I have changed to use your suggestion. Thanks!